bundler/lib/bundler/settings.rb

```diff
@@ -340,6 +372,10 @@ def array_to_s(array)
       array.join(":").tr(" ", ":")
     end
 
+    def redact(password)
+      "*" * password.length
```

Comment:
I can make this a fixed number of `*` to avoid potentially leaking a password length, not sure whether we should care about that.

Reply:
Went ahead and changed it to the fixed string `[REDACTED]`.